tmt-web.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
```

Comment:
This file can be deleted. As we discussed, this will be maintained in Testing Farm deployment (https://gitlab.com/testing-farm/infrastructure/-/merge_requests/570), so here it is a duplicate.